### ISSUE TITLE
wallet_taproot.py fails with "Insufficient funds (-4)"

### ISSUE THREAD

**Issue by MarcoFalke** [bitcoin/bitcoin#25030]
```
 test  2022-04-29T13:16:35.409000Z TestFramework (ERROR): JSONRPC error 
                                   Traceback (most recent call last):
                                     File "/private/var/folders/xx/vl5f934s6k927z1vyyl9cxth0000gn/T/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-apple-darwin/test/functional/test_framework/test_framework.py", line 133, in main
                                       self.run_test()
                                     File "/private/var/folders/xx/vl5f934s6k927z1vyyl9cxth0000gn/T/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-apple-darwin/test/functional/wallet_taproot.py", line 448, in run_test
                                       psbt = self.psbt_online.walletcreatefundedpsbt([], [{self.boring.getnewaddress(): self.psbt_online.getbalance()}], None, {"subtractFeeFromOutputs": [0]})['psbt']
                                     File "/private/var/folders/xx/vl5f934s6k927z1vyyl9cxth0000gn/T/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-apple-darwin/test/functional/test_framework/coverage.py", line 49, in __call__
                                       return_val = self.auth_service_proxy_instance.__call__(*args, **kwargs)
                                     File "/private/var/folders/xx/vl5f934s6k927z1vyyl9cxth0000gn/T/cirrus-ci-build/ci/scratch/build/bitcoin-x86_64-apple-darwin/test/functional/test_framework/authproxy.py", line 144, in __call__
                                       raise JSONRPCException(response['error'], status)
                                   test_framework.authproxy.JSONRPCException: Insufficient funds (-4)
```

https://cirrus-ci.com/task/5522204518514688?logs=ci#L1609

**Comment by michaelfolkson** [comment]
Was this bug fixed?

**Comment by michaelfolkson** [comment]
Fair enough. Where are the high frequent test issues listed? In the good first issues somewhere? Perhaps the situation could be improved if it was clear which ones should be prioritized. (There seems to be at least a couple of existing contributors who are regularly improving the functional tests.) Like everyone I get occasional intermittent issues but I have no idea if they are low frequency or high frequency generally for others on the project or not.

**Comment by michaelfolkson** [comment]
@MarcoFalke: Fixed in #25159?
